conda/wasp/meta.yaml

```diff
@@ -6,7 +6,7 @@
 # ALSO: Follow the directions in scripts/tests/versioner_hashes.yaml
 {% set build = 0 %}
 {% set strbuild = "build_" + build|string %}
-{% set version = "2023.08.17" %}
```

Comment:
I don't believe that we needed to update wasp, but that's fine

Reply:
👍🏻